presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoUninstaller.java

```diff
@@ -52,22 +53,23 @@ public void runCommand()
         }
     }
 
-    private static void uninstallUsingRpm(boolean checkDependencies, boolean ignoreErrors)
+    private static void uninstallUsingRpm(boolean checkDependencies, boolean forceUninstall)
             throws PrestoManagerException
     {
         int checkPackageInstalled = executeCommand("sudo", "rpm", "-q", PRESTO_PACKAGE);
         if (checkPackageInstalled != 0) {
-            if (!ignoreErrors) {
+            if (!forceUninstall) {
                 throw new PrestoManagerException(format("Package '%s' is not installed", PRESTO_PACKAGE), checkPackageInstalled);
             }
             LOGGER.warn("Package '%s' is not installed; Process exited with return value: %s", PRESTO_PACKAGE, checkPackageInstalled);
             return;
         }
         if (executeCommand("service", "presto", "status") == 0) {
-            if (!ignoreErrors) {
+            if (!forceUninstall) {
                 throw new PrestoManagerException("Presto is running");
             }
-            LOGGER.warn("Presto is running, but uninstall will be attempted; Presto will be unstoppable");
+            LOGGER.warn("Presto is running, but uninstall will be attempted");
+            terminatePresto();
```

Comment:
I was indifferent between the options kill or terminate.
Jessica said that terminating presto would be a better option.